.pre-commit-config.yaml

```
@@ -13,7 +13,7 @@ repos:
       - id: trailing-whitespace
         args: [--markdown-linebreak-ext=md]
   - repo: https://github.com/psf/black
```

Swap to the precommit mirror for some added performance.

```suggestion
  - repo: https://github.com/psf/black-pre-commit-mirror
```